README.md

```diff
@@ -48,7 +46,101 @@ embeddings_list = list(embedding_model.embed(documents))
 len(embeddings_list[0]) # Vector of 384 dimensions
 ```
 
-### ⚡️ FastEmbed on a GPU
+Fastembed supports a variety of models for different tasks and modalities.
+
+### 🎒 Dense text embeddings
+
+```python
+from fastembed import TextEmbedding
+
+model = TextEmbedding(model_name="BAAI/bge-small-en-v1.5")
+embeddings = list(embedding_model.embed(documents))
+
+# [
+#   array([-0.1115,  0.0097,  0.0052,  0.0195, ...], dtype=float32),
+#   array([-0.1019,  0.0635, -0.0332,  0.0522, ...], dtype=float32)
+# ]
+
+```
+
+Full list of supported models can be found [here](https://qdrant.github.io/fastembed/examples/Supported_Models/).
```

Comment:
We have this link inside `Dense text embeddings`, and the same link in `Image Embeddings`.

It made me think that this one is a link to the supported dense embeddings, and the latter one is a link to the supported image embeddings, however, both of them are links to the general list with all models.

Reply:
fixed